test/e2e/e2e.go

```diff
@@ -189,7 +189,7 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 			defer func() { _ = f.ClientSet.CoreV1().PersistentVolumes().Delete(pvRoot.Name, &metav1.DeleteOptions{}) }()
 
 			ginkgo.By(fmt.Sprintf("Creating pod to make subpaths /a and /b"))
-			pod := e2epod.MakePod(f.Namespace.Name, nil, []*v1.PersistentVolumeClaim{pvcRoot}, false, "mkdir /mnt/volume1/a && mkdir /mnt/volume1/b")
+			pod := e2epod.MakePod(f.Namespace.Name, nil, []*v1.PersistentVolumeClaim{pvcRoot}, false, "mkdir -p /mnt/volume1/a && mkdir -p /mnt/volume1/b")
```

Comment:
Maybe the test should write to a randomly generated directory name so that we can validate in the test that read/write works?

Reply:
Also how similiar is this test case to https://github.com/kubernetes/kubernetes/blob/5ead4974e0133286b801915b2d8de24bb64e36aa/test/e2e/storage/testsuites/subpath.go#L291?

Reply:
Oh I guess this test case is creating two separate pvs for each subpath

Reply:
Either way I think the test should clean up after itself, i.e. not have weird sideffects like this. I am fine with current form of PR since this part is basically just setting up for the "real " test and if write is failing here then another test should cover that.

Reply:
this test is testing the workaround to issue https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/100, where due to how kubernetes treats volumeHandle , two volumemounts w/ save volumeHandle won't be mounted, so we create pvs with different volumehandles like fs:/a and fs:/b (probably I should have linked it in comment of test)